_compress_images_cache.yml

```diff
@@ -2148,3 +2148,179 @@ assets/images/social-media/social-media-mobile.jpg: social-media-mobile.jpg
 assets/images/social-media/social-media-mobile@2x.jpg: social-media-mobile@2x.jpg
 assets/images/social-media/social-media-preview.jpg: social-media-preview.jpg
 assets/images/profile-picture.jpg: profile-picture.jpg
+assets/images/guide/case-studies/payjoin case study/BIP-78 process.png: BIP-78 process.png
```

Comment:
Could you please remove the outdated cache lines here? Looks like you changed the folder name twice. The cache script is not clever enough to get rid of lines for images that were moved or removed.

Reply:
done.